core/src/test/java/org/apache/iceberg/TestRemoveSnapshots.java

```diff
@@ -1515,4 +1613,51 @@ private RemoveSnapshots removeSnapshots(Table table) {
     RemoveSnapshots removeSnapshots = (RemoveSnapshots) table.expireSnapshots();
     return (RemoveSnapshots) removeSnapshots.withIncrementalCleanup(incrementalCleanup);
   }
+
+  private StatisticsFile writeStatsFileForCurrentSnapshot(
```

Comment:
This name doesn't make sense. The snapshot ID and sequence number are passed in. I think it should be `writeStatsFile`.

Reply:
ACK. Missed renaming during refactoring.